libexec/scoop-checkup.ps1

```diff
@@ -13,6 +13,30 @@ $issues += !(check_windows_defender $true)
 $issues += !(check_main_bucket)
 $issues += !(check_long_paths)
 
+if (!(Test-HelperInstalled -Helper 7zip)) {
+    error "'7-Zip' is not installed! It's required for unpacking most programs. Please Run 'scoop install 7zip' or 'scoop install 7zip-zstd'."
+    $issues++
+}
+
+if (!(Test-HelperInstalled -Helper Lessmsi)) {
+    error "'lessmsi' is not installed! It's required for unpacking programs with MSI packages. Please run 'scoop install lessmsi'."
+    $issues++
+}
+
+if (!(Test-HelperInstalled -Helper Innounp)) {
+    error "'Inno Setup Unpacker' is not installed! It's required for unpacking InnoSetup files. Please run 'scoop install innounp'."
+    $issues++
+}
+
+if (!(Test-HelperInstalled -Helper Dark)) {
+    error "'dark' is not installed! It's require for unpacking installers created with the WiX Toolset. Please run 'scoop install dark' or 'scoop install wixtoolset'."
+    $issues++
+}
+
+if (!(Test-HelperInstalled -Helper Aria2)) {
```

Comment:
Remove this check for aria2.

Forcing users to install aria2 is nonsense.

Reply:
It's a warning and doesn't increase the issue count.

Reply:
```powershell
if (!(Test-HelperInstalled -Helper Lessmsi)) {
    warn "'lessmsi' is not installed. (Optional) This may resolve problems MSI packages. You may run 'scoop install lessmsi'."
}
```

Reply:
I know, but it is irrelevant.

Me, as user who do not want to use aria2, now always see this warning when running checkup. Sure i can intall it and disable it, but it is completely non user friendly.